docs/source/nlp/usage_guides/nlp_properties_extended.rst

```diff
@@ -0,0 +1,85 @@
+calculated for each text sample. The score typically ranges from 0
+(very hard to read, requires intense concentration) to 100 (very easy to read) for english text, though in theory the
+score can range from -inf to 206.835 for arbitrary strings.
+
+Toxicity
+--------
+
+The Toxicity property is a measure of how harmful or offensive a text is. The Toxicity property uses a pre-trained model
+called `unitary/toxic-bert <https://huggingface.co/unitary/toxic-bert>`_ on HuggingFace, which is created and maintained
+by the `Detoxify <https://github.com/unitaryai/detoxify>`_ team, based on the BERT
+architecture and trained on a large corpus of toxic comments. The model assigns a toxicity score to each text,
+ranging from 0 (not toxic) to 1 (very toxic).
+
+Fluency
+-------
+
+The Fluency property is a score between 0 and 1 representing how “well” the input text is written, or how close it is
+to being a sample of fluent English text. A value of 0 represents very poorly written text, while 1 represents perfectly
+written English. The property uses a pre-trained model called
+`prithivida/parrot_fluency_model <https://huggingface.co/prithivida/parrot_fluency_model>`_ on HuggingFace, which
+was created by the authors of the `Parrot Paraphraser <https://github.com/PrithivirajDamodaran/Parrot_Paraphraser>`_
+library.
+
+Formality
+---------
+
+The Formality model returns a measure of how formal the input text is. It uses a pre-traind model called
+`s-nlp/roberta-base-formality-ranker <https://huggingface.co/s-nlp/roberta-base-formality-ranker>`_ on HuggingFace,
+trained by the Skolkovo Institute of Science and Technology (s-nlp).
+The model was trained to predict for English sentences, whether they are formal or informal, where a score of 0
+represents very informal text, and a score of 1 very formal text.
+The model uses the roberta-base architecture, and was trained on
+`GYAFC <https://github.com/raosudha89/GYAFC-corpus>`_ from
+`Rao and Tetreault, 2018 <https://aclanthology.org/N18-1012>`_ and online formality corpus from
+`Pavlick and Tetreault, 2016 <https://aclanthology.org/Q16-1005>`_.
+
+Avoided Answer
+--------------
+
+The Avoided Answer property calculates the probability (0 to 1) of how likely it is that the LLM avoided answering a
+question.
+The property uses a pre-trained bert architecture model that was trained on a dataset of questions and LLM answers
+collected from various LLMs, where the model was trained to predict whether the answer is an avoidance or not.
+
+Grounded in Context
+-------------------q
+
+The Grounded in Context Score is a measure of how well the LLM output is grounded in the context of the conversation,
+ranging from 0 (not grounded) to 1 (fully grounded).
+In the definition of this property, grounding means that the LLM output is based on the context given to it as part of
+the input, and not on external knowledge, for example knowledge that was present in the LLM training data.
```

Comment:
would also mention that this is one kind of hallucination... (essentially making up information it wasn't given, and maybe it has from prior ...)
This can be useful for user searches / general questions